algos.yaml

```diff
@@ -215,7 +215,9 @@ float:
       run-groups:
         fast_pq:
           args: [[1, 2, 3]]
-          query-args: [[1, 2, 3, 4, 6, 8, 10, 13, 16, 20, 24, 28, 33, 38, 44, 50, 57, 64, 72, 80]]
+          # OEIS: A002984
```

Comment:
totally random side topic but maybe E series could be a good choice? https://en.wikipedia.org/wiki/E_series_of_preferred_numbers